MediaManager/MediaManagerBase.cs

```diff
@@ -55,6 +55,7 @@ public virtual IMediaQueue MediaQueue
 
         public virtual void Init()
         {
+            NotificationManager.Enabled = true;
```

Comment:
This should not be here.

Reply:
If I understand good that controls are visible always on notifications screen when player is playing. Therefore on iOS there is nothing about enabling/disabling notifications. You can at least enable/disable background audio but in plist.info

Reply:
moved to the constructor of `NotificationManager`.